scripts/skills/actives/rf_flaming_arrows_skill.nut

```diff
@@ -0,0 +1,219 @@
+this.rf_flaming_arrows_skill <- ::inherit("scripts/skills/actives/aimed_shot", {
+	m = {
+		TargetTile = null,
+		RangeModifier = -2,
```

Comment:
Have you considered just adding this to `this.m.MaxRange` during the `create` function?
We inherit from aimed_shot so we already mimic any changes that mods might do to this value.

Reply:
Unfortunately `this.m.MaxRange` is set by vanilla during `onAfterUpdate`. So we have to do this workaround where we adjust it at the end of `onAfterUpdate` by calling the vanilla `onAfterUpdate` of `aimed_shot`.